lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -777,7 +777,17 @@ module Resources
             'start': '2012-05-17T15:33:18.000Z'
           }
         }
-      }
+      },
+      {
+        'op': 'test',
+        'path': '/name/0/id',
+        'value': 'CI-12782404-0'
```

Comment:
the example patch is calling for RP E-12467982-97731398
so i think the id should be CI-12467982-0

Reply:
https://github.com/cerner/fhir.cerner.com/commit/8bbec6c06cc084c792e14d3c466bf4f299336cfa#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR786